easybuild/scripts/bootstrap_eb.py

```diff
@@ -240,7 +240,7 @@ def check_module_command(tmpdir):
 
     def check_cmd_help(modcmd):
         """Check 'help' output for specified command."""
-        modcmd_re = re.compile(r'module\s.*command\s')
+        modcmd_re = re.compile(r'module\s.*command')
```

Comment:
Note to self: the previous regex was slightly too strict for the output of `modulecmd.tcl python help` produced by Modules v4, but still good enough to check that the command works in general imho.